flake.nix

```diff
@@ -21,6 +21,9 @@
         pkgs = import nixpkgs.outPath {
           config = {
             allowUnfree = true;
+            permittedInsecurePackages = [
+              "openssl-1.1.1u"
```

Comment:
This may not work when overriding `nixpkgs` input. And when using the overlay. But perhaps the latter is fine.

Reply:
:thinking: are you sure? Shouldn't this work since this imports nixpkgs via `.outPath`?

Reply:
Sorry, I meant that if you override `nixpkgs`, there might be a different version of openssl.

Reply:
Ah yeah, makes sense, that could happen.